Assignment 10

hhan-solvd opened the pull request:
Added a new enum, reformatted enums, and added streams and reflections.

hhan-solvd committed:
Updated CustomerType.java

Reformatted the file

hhan-solvd committed:
Updated EmployeeTitle.java

Reformatted the file

hhan-solvd committed:
Update EmployeeType.java

Reformatted the file

hhan-solvd committed:
Updated VehicleBodyStyle.java

Reformatted the file

hhan-solvd committed:
Added PaymentMethod.java

hhan-solvd committed:
Updated Finance.java

Updated with enum

hhan-solvd committed:
Updated Dealership.java

Added stream

hhan-solvd committed:
Added Reflection

hhan-solvd committed:
Updated Main.java

Added stream and reflection

hhan-solvd committed:
Updated Reflection.java

Updated file with logger

ankochergin commented on src/main/java/com/solvd/app/services/Finance.java:
```diff
@@ -27,11 +28,11 @@ public double getLoanAmount() {
     public void determinePaymentMethod() {
         if (customer.getBudget() < vehicle.getPrice()) {
-            paymentMethod = "Finance";
+            paymentMethod = PaymentMethod.FINANCE.getPaymentMethod();
```
```suggestion
            paymentMethod = PaymentMethod.FINANCE;
```

ankochergin replied:
change your `private String paymentMethod;` variable to `private PaymentMethod paymentMethod;`

hhan-solvd committed:
Update src/main/java/com/solvd/app/services/Finance.java

Co-authored-by: ankochergin <103670717+ankochergin@users.noreply.github.com>

hhan-solvd committed:
Updated Finance.java

Changed the type of the field "paymentMethod" from String to PaymentMethod, and updated getPaymentMethod() and determinePaymentMethod() accordingly.

hhan-solvd committed:
Update and rename Reflection.java to ReflectionUtils.java

hhan-solvd committed:
Updated Main.java

Rename Reflection.java to ReflectionUtils.java
